### PR TITLE
*: update to the preview3

### PR DESCRIPTION
**Pull request by fyrchik**
Update both Go and C# to preview3 version.

### CODE REVIEW

File: .docker/build/Dockerfile.sharp [hunk 1]
```diff
@@ -44,7 +46,7 @@ RUN wget -O /opt/neo-cli.zip ${URL} && \
     cp /temp/neo-cli/Microsoft.AspNetCore.ResponseCompression.dll /neo-cli && \
```
Comment (roman-khimov) [inline .docker/build/Dockerfile.sharp:46]: It shouldn't be needed now.

File: .docker/build/Dockerfile.golang [hunk 2]
```diff
@@ -6,7 +6,7 @@ WORKDIR /neo-go
 # Install deps:
 RUN apk add --no-cache git
 
-ARG REV="v0.90.0"
+ARG REV="master"
```
Comment (im-kulikov) [inline .docker/build/Dockerfile.golang:9]: bad idea because makes it impossible to reproduce at different times

Comment (fyrchik) [reply .docker/build/Dockerfile.golang:9]: Yeah, I will change it right after `0.91.0` is out.

Comment (roman-khimov) [reply .docker/build/Dockerfile.golang:9]: I think we want to set it to some recent commit for now (to merge it correctly) and then of course change to 0.91.0 when it's out.

File: .docker/build/Dockerfile.sharp (hunk 1 above)
Comment (roman-khimov) [reply .docker/build/Dockerfile.sharp:46]: But it's still there, any reason for that?

File: .docker/ir/go.protocol.privnet.one.yml [hunk 3]
```diff
@@ -1,8 +1,9 @@
 ProtocolConfiguration:
   Magic: 56753
-  MemPoolSize: 5000
+  MemPoolSize: 500000
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply .docker/ir/go.protocol.privnet.one.yml:3]: As this one too.

File: .docker/build/Dockerfile.sharp [hunk 4]
```diff
@@ -44,7 +46,7 @@ RUN wget -O /opt/neo-cli.zip ${URL} && \
...
     rm -r /temp
 
-ENV MODULES="RpcServer SystemLog"
+ENV MODULES="RpcServer"
```
Comment (roman-khimov) [inline .docker/build/Dockerfile.sharp:49]: And BTW, this probably requires some changes to C# node configuration (see default configs there).

Comment (fyrchik) [reply .docker/build/Dockerfile.sharp:49]: Fixed.

File: .docker/build/Dockerfile.golang (hunk 2 above)
Comment (fyrchik) [reply .docker/build/Dockerfile.golang]: Fixed.

**Pull request merged**